security/ldap.rst

```diff
@@ -382,5 +404,66 @@ Configuration example for HTTP Basic
             ),
         );
 
+Configuration example for form login and query_string
+.....................................................
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/security.yml
+        security:
+            # ...
+
+            firewalls:
+                main:
+                    # ...
+                    form_login_ldap:
+                        login_path: login
+                        check_path: login_check
```

Comment:
Recently we updated the docs to use the same route for `login_path` and `check_path`. I think we should do that here too.